kuma/javascript/src/last-modified.jsx

```diff
@@ -30,9 +30,9 @@ export default function LastModified({
     return (
         <section className="contributors-sub">
             <ClockIcon />
-            <header>
-                <h4>{gettext('Last modified:')}</h4>
-            </header>
+            <span className="contributors-sub-label">
+                {gettext('Last modified:')}
+            </span>
```

Comment:
We should not change this HTML structure as it is specifically done for semantic and accessibility reasons. I will also add the CSS to resolve this problem in a separate comment.